lighthouse-core/test/chromium-web-tests/test-page-devtools.sh

```diff
@@ -30,30 +32,58 @@ bash "$SCRIPT_DIR/download-content-shell.sh"
 
 bash "$SCRIPT_DIR/roll-devtools.sh"
 
-# Add test to run lighthouse in DevTools and print LHR.
-echo "
-(async function() {
-  await TestRunner.navigatePromise('$1');
+WEB_TEST_DIR="$DEVTOOLS_PATH/test/webtests/http/tests/devtools/lighthouse-run"
+
+# Remove any tests leftover from previous run.
+if [ -d "$WEB_TEST_DIR" ]; then
+  rm -rf "$WEB_TEST_DIR"
+fi
+mkdir -p "$WEB_TEST_DIR"
+
+echo "Creating test files for URLs"
+COUNTER=0
+while read url; do
+  [[ -z $url || ${url:0:1} == "#" ]] && continue
```

Comment:
nice idea with the comment

Reply:
It was necessary for our example urls.txt

Reply:
I'm not gonna lie @connorjclark 
[![image](https://user-images.githubusercontent.com/2301202/101409939-39c5fa00-38a4-11eb-9972-c8e56ffe1d3d.png)](https://github.com/GoogleChrome/lighthouse/blame/master/lighthouse-core/scripts/gcp-collection/gcp-run.sh#L33)

[Also](https://www.youtube.com/watch?v=XzbAEHdy8oU)